xmodule.grace

```diff
@@ -220,12 +220,22 @@ method checkimport(nm, pathname, line, linePos, isDialect) is confidential {
             } elseif { binaryFile.newer(moduleFileGrace).not } then {
                 util.log 60 verbose "{binaryFile} not newer than {moduleFileGrace}"
             }
-            compileModule (nm) inFile (moduleFileGrace.asString) 
+            compileModule (nm) inFile (moduleFileGrace.asString)
                 forDialect (isDialect) atRange (line, linePos)
         }
         importsSet.add(nm)
     } elseif { util.target == "js" } then {
         def moduleFileJs = moduleFileGct.copy.setExtension ".js"
+        if (moduleFileJs.exists) then {
```

Comment:
Do we want to leave these logging messages here?  If we are getting repeated issues, it might be a good idea, but 
if you have solved it, perhaps we should take them out?